portfolio/src/main/webapp/script.js

```diff
@@ -77,3 +77,145 @@ function createListElement(text) {
   liElement.innerText = text;
   return liElement;
 }
+
+class Place {
+  constructor(name, latitude, longitude, year, message = "") {
+    this.name = name;
+    this.latitude = latitude;
+    this.longitude = longitude;
+    this.year = year;
+    this.message = message;
+  }
+}
+
+class PlaceBuilder{
+  constructor(name) {
+    this.name = name;
+    return this;
+  }
+  withLatitude(latitude) {
+    this.latitude = latitude;
+    return this;
+  }
+  withLongitude(longitude) {
+    this.longitude = longitude;
+    return this;
+  }
+  withYear(year) {
+    this.year = year;
+    return this;
+  }
+  withMessage(message) {
+    this.message = message;
+    return this;
+  }
+  build() {
+    return new Place(this.name, this.latitude, this.longitude, this.year, this.message);
+  }
+}
+
+
+
+/** Places I have lived. */
+const placesLived = [
+  new PlaceBuilder("Evanston").withLatitude(42.0451).withLongitude(-86.6877).withYear('2000-2002')
+    .withMessage('I was born in outside Chicago at Northwestern Medical Center.').build(),
+  new PlaceBuilder("Overland Park").withLatitude(38.9822).withLongitude(-94.6708).withYear('2002-2017')
+    .withMessage('I grew up in the Kansas City suburbs, first in Lenexa and then Overland Park. I attended Blue Valley North high school for three years.').build(),
```

Comment:
[Nit] With template literals you can split this string into multiple lines by using backticks.  That way you can keep each line under 80 characters.

See https://stackoverflow.com/questions/805107/creating-multiline-strings-in-javascript for details.

Same applies to lines 126 and 130.